pkgs/applications/editors/emacs/elisp-packages/manual-packages/elpaca/default.nix

```diff
@@ -0,0 +1,24 @@
+{
+  melpaBuild,
+  fetchFromGitHub,
+  git,
+}:
+
+melpaBuild {
+  pname = "elpaca";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "progfolio";
+    repo = "elpaca";
+    rev = "db2fd7258ff69fe2d100888cb8d92cf3bf94d465";
+    hash = "sha256-SseY0iU3D3cloKZy6xPp8QT0H1Cu2uGiiVG6rXq/UHg=";
+  };
+
+  packageRequires = [ git ];
+
+  meta = {
```

Comment:
- `meta.license` is missing.
- `meta.maintainers` is missing.  Packages with a missing or empty `meta.maintainers` cannot be added to Nixpkgs.  If you want to become its maintainer, add yourself to the maintainer list in the first commit and put the current commit as the second one.  doc: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md#how-to-become-a-maintainer.